### DISCUSSION TITLE
[External] [Bug] Ability to submit to SRA without ref_gff and ref_fasta?

### DISCUSSION THREAD

**Discussion by mollymitchell70** [CDCgov/tostadas#34] [Issues] [answered]
**What is the Bug Related To? Please Provide a Description.**&nbsp;  
Hello, I am trying to submit only to SRA using .fastq.gz. I am getting an error relating to empty fields for ref_fasta and ref_gff. Is there a way to do this?

**Categorize the Severity of the Bug from 1-5:** 3
* 1 = most severe (a full-break in core function)
* 2-4 = moderate (break for a particular aspect/feature) (how integral is the broken feature?)
* 5 = least severe (non-functional issue, such as inconsitency / error in documentation or administrative in nature)&nbsp;   

**Please Complete the Following Information:**
 - OS: [e.g. iOS]: Linux
 - Browser [e.g. chrome, safari]: Chrome
 - Version [e.g. 22]: 22
 - Run environment (container, cloud service, HPC, platform, etc.): HPC, conda

 **Please Outline Necessary Steps to Replicate Bug (Go to.. Click on... Install the following... etc.):**
Command: 
nextflow run main.nf -profile standard,conda -entry only_initial_submission -params-file $PWD/nf_params/standard.yml 
Error: 
(/blue/bphl-florida/mollymitchell/conda_envs/tostadas) [mollymitchell@c0700a-s27tostadas]$ nextflow run main.nf -profile standard,conda -entry only_initial_submission -params-file $PWD/nf_params/standard.yml
N E X T F L O W  ~  version 22.10.6
WARNING: Could not load standard_params.config from /blue/bphl-florida/mollymitchell/tostadas/tostadas/nf_params/conf/
Launching `main.nf` [zen_stone] DSL2 - revision: 892ec7b297
executor >  local (1)
executor >  local (1)
[a7/49f8e3] process > only_initial_submission:SUB... [100%] 1 of 1 â
[-        ] process > only_initial_submission:PRE... -
[-        ] process > only_initial_submission:SUB... -
Error executing process > 'only_initial_submission:PREP_SUBMISSION_ENTRY'

Caused by:
  Not a valid path value: 'null'

Tip: you can replicate the issue by changing to the process work dir and entering the command `bash .command.run`

**Any Additional Context or Information? Has There Been Any Progress Made So Far Towards this Request? Any Concrete Instructions to Resolve the Bug or Helpful Resources to Reference? Screenshots or Logs?**

**Comment by ankushkgupta2** [comment, marked as answer]
@mollymitchell70  Thanks for submitting the issue. 

I am looking into it and noticed the following in the error message: 

```"WARNING: Could not load standard_params.config from /blue/bphl-florida/mollymitchell/tostadas/tostadas/nf_params/conf/"```

This is because it is attempting to read in the standard_params.config within the conf directory instead of the params file you are passing in. It should be resolved after removing the "standard" after -profile:

```bash
nextflow run main.nf -profile conda -entry only_initial_submission -params-file $PWD/nf_params/standard.yml 
```

Using this should resolve the issue and you should be able to run the only_initial_submission entrypoint without a reference fasta/gff. Only a validated metadata file and fasta file would be needed.
